src/components/GameRoom.js

```diff
@@ -137,57 +134,65 @@ function GameRoom({ room, setInRoom, userName, host, gamesWon,_id }) {
     setInRoom(false);
   };
 
-
-
   const disconnectRoom = () => {
     socket.emit("leave_room", room);
     socket.disconnect();
     setInRoom(false);
   };
 
-  function wordHandler(event){
-    const {value} = event.target
-    setWord(value)
+  function wordHandler(event) {
+    const { value } = event.target;
+    setWord(value);
   }
 
-  function newGame(){
-    setGameOver(false)
-    setGameStarted(false)
-    setGuessingYourWord(false)
-    setAllPlayersReady(false)
-    setYouWon(false)
+  function newGame() {
+    setGameOver(false);
+    setGameStarted(false);
+    setGuessingYourWord(false);
+    setAllPlayersReady(false);
+    setYouWon(false);
   }
 
   let guess = youGuessed ? "You Guessed Right!!!" : "";
 
   //Disables starting a new game/new round unless all players are ready and there are at least a minimum of 3 players
-  let dis = players.length > 2 && allPlayersReady ? false : true;  
+  let dis = players.length > 2 && allPlayersReady ? false : true;
 
   return (
     <div>
       <div style={logoffStyle}>
-        <h2>{userName}</h2>
+        <h2><span style={{color:"#ECBE07"}}>{userName}</span> &nbsp;  Room: <span style={{color:"#ECBE07"}}>{room}</span></h2>
         <button onClick={disconnectRoom}>Logout</button>
-        {/* <select id="navOptions" onChange={hamburgerNav}>
-          <option value="">Hamburger nav placeholder</option>
-          <option value="option1">Profile Page</option>
-          <option value="option2">Game Lobby</option>
-        </select> */}
       </div>
-
       <div>
-        <h1>You are in Room {room}</h1>
-        <h2>Current players are: {players.join('-')}</h2>
-        <button onClick={leaveRoom}>Leave Room</button>        
+        <h2>
+          <strong>Current players are:</strong>{" "}
+        </h2>
+        <ul>
+          {players.map((player) => (
+            <div key={player}>
+              <li
+                style={{
+                  marginLeft: "45%",
+                  textAlign: "left",
+                }}
+              >
+                - {player}
+              </li>
+            </div>
+          ))}
+        </ul>
+        {console.log("players", players)}
+        <button onClick={leaveRoom}>Leave Room</button>
+        <button onClick={disconnectRoom}>Disconnect</button>
```

Comment:
The button disconnect is not necessary any more, now we have the Logout btn (line 165 this same file)